android/src/main/java/com/reactnativecommunity/asyncstorage/ReactDatabaseSupplier.java

```diff
@@ -85,7 +86,13 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
         if (tries > 0) {
           deleteDatabase();
         }
-        mDb = getWritableDatabase();
+
+        if(disableLocalizedCollators) {
```

Comment:
Do we really need the class member?
```suggestion
        if (BuildConfig.AsyncStorage_noLocalizedCollators) {
```

Reply:
True, nice catch